### ISSUE TITLE
[gui error report] Error: Command failed: powershell.exe -NoProfile -NonInteractive -InputF

### ISSUE THREAD

**Issue by Destroyinator69420** [ipfs/ipfs-desktop#2335]
👉️ Please describe what you were doing when this error happened.

**Specifications**

- **OS**: win32 10.0.22000
- **IPFS Desktop Version**: 0.24.0
- **Electron Version**: 19.0.9
- **Chrome Version**: 102.0.5005.167

**Error**

```
Error: Command failed: powershell.exe -NoProfile -NonInteractive -InputFormat None -Command Get-AuthenticodeSignature -LiteralPath 'C:\Users\Jknap\AppData\Local\ipfs-desktop-updater\pending\temp-IPFS-Desktop-Setup-0.24.1.exe' | ConvertTo-Json -Compress | ForEach-Object { [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($_)) }
Get-AuthenticodeSignature : File 
C:\Users\Jknap\AppData\Local\ipfs-desktop-updater\pending\temp-IPFS-Desktop-Setup-0.24.1.exe was not found.
At line:1 char:1
  Get-AuthenticodeSignature -LiteralPath 'C:\Users\Jknap\AppData\Local\ ...
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      CategoryInfo          : ObjectNotFound: (:) [Get-AuthenticodeSignature], FileNotFoundException
      FullyQualifiedErrorId : SignatureCommandsBaseFileNotFound,Microsoft.PowerShell.Commands.GetAuthenticodeSignature 
   Command
 

    at ChildProcess.exithandler (node:child_process:408:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
```

**Comment by SgtPooki** [comment]
@Destroyinator69420 Thanks for submitting this issue. I'm not sure we have enough information to help you out. Can you please provide more information:

* what were you doing when the error occurred?
* what are you trying to accomplish by running it via PowerShell? or was it done by some third-party tool / process?

It looks like https://blog.doyensec.com/2020/02/24/electron-updater-update-signature-bypass.html and https://github.com/signalapp/Signal-Desktop/issues/3059 provide some context into the reason we're running into this issue, but can you provide any more details?

**Comment by beyondkmp** [comment]
We can replace the PowerShell with [a native verify signature module](https://github.com/beyondkmp/win-verify-trust) to resolve all powershell problems.

https://github.com/electron-userland/electron-builder/pull/7337

**Comment by lidel** [comment]
Sounds like https://github.com/electron-userland/electron-builder/issues/7127 got fixed (https://github.com/electron-userland/electron-builder/issues/7127#issuecomment-1991876763) and (hopefully) will  be fixed by electron-updater update done in #1856
